intranet/templates/eighth/emails/students_stickied.html

```diff
@@ -0,0 +1,16 @@
+<html>
+  <body>
+    {% with description="Eighth Signup Changes" %}
+        {% include "email_metadata.html" %}
+    {% endwith %}
+
+    <p>
+      You have been stickied into {{ activity.activity.name }} on {{ activity.date }} ({{ activity.block_letter }}).
+
+      If there has been a mistake with this change, please contact the 8th period office.
+      Have a nice day!
```

Comment:
I dont think you need this lol

Reply:
It never hurts to tell people to have a nice day :)